examples/nodejs-client/index.js

```diff
@@ -0,0 +1,16 @@
+const {HawkTracerClient} = require('@hawktracer/client')
```

Comment:
Could we add `try...catch` to this example so user can see where the pottential error can happen?

Reply:
Yes I missed that.